### PR TITLE
Feat/worker timer

### LINKED ISSUE #138: Make timer work on mobile phones
JavaScript will stop at some time to refresh the timer or play the sound when the phone is in sleep mode.

### PR DESCRIPTION
**Pull request by olastor**
Probably closes #138

### CODE REVIEW

File: src/app/meditation/meditation.component.ts [hunk 1]
```diff
@@ -132,16 +133,6 @@ export class MeditationComponent {
           );
         }
 
-        // also checking here if walking or sitting finished for the current user
-        // to play a sound. Doing it inside the filter to reduce iterations.
-        if (data._id === this.currentMeditation && this.userWalking && !data.walkingLeft){
-          this.userWalking = false;
```
Comment (binarious) [inline src/app/meditation/meditation.component.ts]: We still need `userWalking` and `userSitting`.

File: package.json [hunk 2]
```diff
@@ -87,6 +87,7 @@
     "rxjs": "5.0.0-beta.6",
     "scriptjs": "^2.5.8",
     "socket.io-client": "^1.4.6",
+    "worker-timer": "^1.1.0",
```
Comment (binarious) [inline package.json]: Can you swap it with `worker-timers`?

Comment (olastor) [inline package.json]: Yes, haven't tested it, though

Comment (binarious) [inline package.json]: Can you do with 1m + 1m?

**Pull request merged**